service/controller/resource/clusterconfigmap/resource.go

```diff
@@ -42,7 +42,7 @@ type Resource struct {
 // New creates a new configured config map state getter resource managing
 // cluster config maps.
 //
-//     https://godoc.org/github.com/giantswarm/operatorkit/v2/pkg/resource/k8s/secretresource#StateGetter
+//     https://godoc.org/github.com/giantswarm/operatorkit/v4/pkg/resource/k8s/secretresource#StateGetter
```

Comment:
This link does not work

Reply:
Neither does the v2 one :shrug:

Not sure what to do here.